wfl/Controllers/MenuSelectionsController.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Web.Http;
+using System.Threading.Tasks;
+using System.Linq;
+using System.Collections.Generic;
+using Symphono.Wfl.Models;
+using Symphono.Wfl.Database;
+
+namespace Symphono.Wfl.Controllers
+{
+    [RoutePrefix("api/food-order/{foodOrderid}/menu-selection")]
+    public class MenuSelectionsController : ApiController
+    {
+        private IDBManager dbManager { get; }
+        public MenuSelectionsController(IDBManager dbManager) {
+           this.dbManager = dbManager;
+        }
+
+        [Route("")]
+        [HttpGet]
+        public async Task<IHttpActionResult> GetAsync([FromUri] string foodOrderId)
+        {
+            FoodOrder order = await dbManager.GetEntityByIdAsync<FoodOrder>(foodOrderId);
+            return Ok(order.MenuSelections);
+        }
+
+        [Route("{index}")]
+        [HttpGet]
+        public async Task<IHttpActionResult> GetByIndexAsync([FromUri] string foodOrderId, [FromUri] int index)
```

Comment:
This goes along with the delete menu selection PR:  I would not use the MenuSelections positional index as a factor for getting/deleting. MenuSelection should have an identifier like every other Model object and should be referred to by that identifier. So basically I would get rid of index and everywhere you are using index use id instead.